st2common/st2common/util/param.py

```diff
@@ -150,8 +152,32 @@ def _render(node, render_context):
     Render the node depending on its type
     '''
     if 'template' in node:
+        complex_type = False
+
+        LOG.debug("Rendering param of type: %s", type(node['template']))
+
+        if isinstance(node['template'], list) or isinstance(node['template'], dict):
+            node['template'] = json.dumps(node['template'])
```

Comment:
Just thinking out loud.

I assume this will always work since `node['template']` can only contain simple types which are JSON serializable, right?

Because if not, this could cause issues with types which are not JSON serializable (datetime, class instances, etc).